src/app/service/poap/EndPOAP.ts

```diff
@@ -31,7 +31,7 @@ export default async (guildMember: GuildMember, code?: string, ctx?: CommandCont
 		throw new ValidationError(`<@${guildMember.id}> Hmm it doesn't seem you are hosting an active event.`);
 	}
 	
-	await ServiceUtils.tryDMUser(guildMember, 'Hello, I\'m really enjoying the event...');
+	await ServiceUtils.tryDMUser(guildMember, 'Finally it\'s over...');
```

Comment:
Maybe we give this a little more hype. The tone kind of makes it seems like "I couldn't wait for this to be over ..." 😂
Maybe something like ... "That was a great event! Now to send out some POAPs." or something like that?

Reply:
lol yeah that was the intention since DEGEN is busy checking everyone coming and going I figured we can make it seem like DEGEN is tired, what do you think?

Reply:
Ah gotcha. I like that. Maybe something like "Whew, that was a lot of work ..." ?

Reply:
Updated!